.github/workflows/format-test-build.yml

```
@@ -0,0 +1,97 @@
+name: PR-Workflow
+on: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)